tests/test_highlighting.py

```diff
@@ -14,6 +14,11 @@
 from sphinx.highlighting import PygmentsBridge
 
 if TYPE_CHECKING:
+    from collections.abc import Iterable
+    from typing import Never, TextIO
+
+    from pygments.token import _TokenType
```

Comment:
Should we be using a private name from a third-party library?

Reply:
I don't feel strongly. The downside of this is that it might break in future, but at least it is only a test file (so no surprise user breakages). The best alternative, I believe, is `tuple[typing.Any, str]`. Do you have a preference?